src/azul/__init__.py

```diff
@@ -1822,17 +1824,14 @@ def __attrs_post_init__(self):
                                          period=10 * 60,
                                          retry_after=30)
 
-    #: The rate limit for file download requests
+    #: The rate limit for file download requests.
     #:
-    #: We aim for a global limit of 60 file downloads per 10 minutes. Based on
-    #: an observed average of 2.9 distinct IPs concurrently downloading files
-    #: in any 10-minute window, the maximum per-IP request rate we can allow is
-    #: 20/10min, or 10/5min.
+    #: Currently this is set to the same value as the global rate limit, which
+    #: makes the WAF rule driven by this property redundant, however we keep
+    #: this property to distinguish the rate limits we want to enforce.
```

Comment:
This is uninformative and awkwardly put. You don't need to tell anyone that the limits are equal, or why we keep both attributes around. Both are be readily apparent from the code. This is the piece of information that is not obvious:

```suggestion
    #: Now that most files are mirrored so we can serve them at the general rate.
```